libcontainer/configs/validate/rootless_test.go

```diff
@@ -98,84 +76,6 @@ func TestValidateRootlessMappingGid(t *testing.T) {
 	if err := validator.Validate(config); err == nil {
 		t.Errorf("Expected error to occur if no gid mappings provided")
 	}
-
-	config = rootlessConfig()
-	config.GidMappings[0].HostID = getegid() + 1
-	if err := validator.Validate(config); err == nil {
-		t.Errorf("Expected error to occur if getegid() != mapped gid")
-	}
-
-	config = rootlessConfig()
-	config.GidMappings[0].Size = 1024
-	if err := validator.Validate(config); err == nil {
-		t.Errorf("Expected error to occur if more than one gid mapped")
-	}
-
-	config = rootlessConfig()
-	config.GidMappings = append(config.GidMappings, configs.IDMap{
-		HostID:      getegid() + 1,
-		ContainerID: 0,
-		Size:        1,
-	})
-	if err := validator.Validate(config); err == nil {
-		t.Errorf("Expected error to occur if more than one gid extent mapped")
-	}
-}
-
-/* rootlessMount() */
-
-func TestValidateRootlessMountUid(t *testing.T) {
-	config := rootlessConfig()
-	validator := New()
-
-	config.Mounts = []*configs.Mount{
-		{
-			Source:      "devpts",
-			Destination: "/dev/pts",
-			Device:      "devpts",
-		},
-	}
-
-	if err := validator.Validate(config); err != nil {
-		t.Errorf("Expected error to not occur when uid= not set in mount options: %+v", err)
-	}
-
-	config.Mounts[0].Data = "uid=5"
-	if err := validator.Validate(config); err == nil {
-		t.Errorf("Expected error to occur when setting uid=5 in mount options")
-	}
-
-	config.Mounts[0].Data = "uid=0"
-	if err := validator.Validate(config); err != nil {
-		t.Errorf("Expected error to not occur when setting uid=0 in mount options: %+v", err)
-	}
-}
-
-func TestValidateRootlessMountGid(t *testing.T) {
-	config := rootlessConfig()
-	validator := New()
-
-	config.Mounts = []*configs.Mount{
-		{
-			Source:      "devpts",
-			Destination: "/dev/pts",
-			Device:      "devpts",
-		},
-	}
-
-	if err := validator.Validate(config); err != nil {
-		t.Errorf("Expected error to not occur when gid= not set in mount options: %+v", err)
-	}
-
-	config.Mounts[0].Data = "gid=5"
-	if err := validator.Validate(config); err == nil {
-		t.Errorf("Expected error to occur when setting gid=5 in mount options")
-	}
-
-	config.Mounts[0].Data = "gid=0"
-	if err := validator.Validate(config); err != nil {
-		t.Errorf("Expected error to not occur when setting gid=0 in mount options: %+v", err)
-	}
```

Comment:
... so all of these removed tests should stay, and we should add new tests making sure that multiple-mapping cases are still treated sanely.